bot/cogs/moderation.py

```diff
@@ -181,12 +198,137 @@ async def clear(self, ctx: Context, amount: int) -> None:
                 **Amount**: {amount}
                 """
             ),
-            color=discord.Color.orange(),
+            color=Color.orange(),
         )
         message = await ctx.send(ctx.author.mention, embed=embed)
         await asyncio.sleep(2.5)
         await message.delete()
 
+    @command()
+    @has_permissions(manage_messages=True)
+    async def shift(self, ctx, count: int, target: TextChannel, copy: bool = False) -> None:
+        """Copy or Move specified messages amount to specified channel"""
+        messages = []
+        async for message in ctx.channel.history(limit=count):
+            embed = Embed(description=message.content, color=Color.green())
+            embed.set_author(name=message.author.name, icon_url=message.author.avatar_url)
+            embed.timestamp = message.created_at
+            messages.append(embed)
+
+            if not copy:
+                await message.delete()
+
+        await target.send(f'Message source : {ctx.channel.mention}.')
+
+        for embed in reversed(messages):
+            await target.send(embed=embed)
```

Comment:
```suggestion
        for embed in reversed(messages):
            await target.send(embed=embed)
            await asyncio.sleep(0.5)
```
again, to prevent or at least delay rate-limit staggering